addons/mail/static/src/components/activity_box/activity_box.xml

```diff
@@ -3,37 +3,37 @@
-            <t t-if="chatter and chatter.thread">
-                <a role="button" class="o_ActivityBox_title btn" t-on-click="chatter.onClickActivityBoxTitle">
+            <t t-if="activityBoxView and activityBoxView.chatter">
+                <a role="button" class="o_ActivityBox_title btn" t-on-click="activityBoxView.onClickActivityBoxTitle">
                     <hr class="o_ActivityBox_titleLine" />
                     <span class="o_ActivityBox_titleText">
-                        <i class="fa fa-fw" t-att-class="chatter.isActivityBoxVisible ? 'fa-caret-down' : 'fa-caret-right'"/>
+                        <i class="fa fa-fw" t-att-class="activityBoxView.isActivityBoxVisible ? 'fa-caret-down' : 'fa-caret-right'"/>
                         Planned activities
                     </span>
-                    <t t-if="!chatter.isActivityBoxVisible">
+                    <t t-if="!activityBoxView.isActivityBoxVisible">
                         <span class="o_ActivityBox_titleBadges">
-                            <t t-if="chatter.thread.overdueActivities.length > 0">
+                            <t t-if="activityBoxView.chatter.thread.overdueActivities.length > 0">
                                 <span class="o_ActivityBox_titleBadge badge rounded-circle badge-danger">
-                                    <t t-esc="chatter.thread.overdueActivities.length"/>
+                                    <t t-esc="activityBoxView.chatter.thread.overdueActivities.length"/>
                                 </span>
                             </t>
-                            <t t-if="chatter.thread.todayActivities.length > 0">
+                            <t t-if="activityBoxView.chatter.thread.todayActivities.length > 0">
                                 <span class="o_ActivityBox_titleBadge badge rounded-circle badge-warning">
-                                    <t t-esc="chatter.thread.todayActivities.length"/>
+                                    <t t-esc="activityBoxView.chatter.thread.todayActivities.length"/>
                                 </span>
                             </t>
-                            <t t-if="chatter.thread.futureActivities.length > 0">
+                            <t t-if="activityBoxView.chatter.thread.futureActivities.length > 0">
                                 <span class="o_ActivityBox_titleBadge badge rounded-circle badge-success">
-                                    <t t-esc="chatter.thread.futureActivities.length"/>
+                                    <t t-esc="activityBoxView.chatter.thread.futureActivities.length"/>
                                 </span>
                             </t>
                         </span>
                     </t>
                     <hr class="o_ActivityBox_titleLine" />
                 </a>
-                <t t-if="chatter.isActivityBoxVisible">
+                <t t-if="activityBoxView.isActivityBoxVisible">
                     <div class="o_ActivityList">
```

Comment:
(not related to your diff, but I just noticed the name of this part is wrong, it should be `o_ActivityBox_activityList` instead)